map/hashtable.py

```diff
@@ -13,8 +14,8 @@ class HashTable(object):
     in Return True for a statement of the form key in map, if the given key is in the map, False otherwise.
     """
 
-    _empty = object()
-    _deleted = object()
+    _empty = None
+    _deleted = None
```

Comment:
`_empty` and `_deleted` act as placeholders for not yet assigned keys and deleted keys respectively. 
In many places, there are uses like  `self.keys[hash_] is _empty` and `self.keys[hash_] is _deleted` which won't work as expected if be replace them both by `None`.

Reply:
@goswami-rahul  I ran all tests successful. After each change I ran the tests and it seems no problem.